alias.txt

```diff
@@ -69,6 +69,8 @@
 .sqc Squawk Charlie.
 .sqs Squawk Standby.
 .id Identified.
+.idT Identified, please remember this standard tip when you fly as Text-Pilot, fly first all written manoeuvres before you write back the instruction thanks

```

Comment:
These kind of hins for pilots on frequency is not nice according to CoC A16, this should be done in private chat instead. I would vote to remove this line.